src/com/lftechnology/java/training/alina/jdbc/service/DateTimeService.java

```diff
@@ -0,0 +1,15 @@
+package com.lftechnology.java.training.alina.jdbc.service;
+
+import java.sql.Timestamp;
+
+/**
+ * DateTimeService consists of date and time related functionalities
+ * 
+ * @author Alina Shakya <alinashakya@lftechnology.com>
+ */
+public class DateTimeService {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)